docs/content/how-to/kubevirt/external-infrastructure.md

```diff
@@ -59,6 +59,7 @@ The user or service account used in the provided kubeconfig should have full per
 * `virtualmachineinstances.kubevirt.io`
 * `datavolumes.cdi.kubevirt.io`
 * `services`
+* `endpointslices`
```

Comment:
I think the `restricted` subresource needs to be documented too, right?

Reply:
Right will add it.